posthog/api/test/test_feature_flag.py

```diff
@@ -348,28 +350,194 @@ def test_updating_a_feature_flag_with_same_team_and_key_of_a_deleted_one(self):
+        # # alpha-feature is set for "distinct_id"
+        distinct_id_user = User.objects.create_and_join(self.organization, "distinct_id_user@posthog.com", None)
+        distinct_id_user.distinct_id = "distinct_id"
+        distinct_id_user.save()
+        self.client.force_login(distinct_id_user)
+        response = self.client.get("/api/feature_flag/my_flags")
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(response.json()["distinct_id"], "distinct_id")
-        self.assertEqual(sorted(response.json()["flags_enabled"]), ["alpha-feature", "red_button"])
+        self.assertEqual(response.json()["flags"], {"alpha-feature": "third-variant", "red_button": True})
 
         # alpha-feature is not set for "distinct_id_0"
-        response = self.client.get("/api/feature_flag/for_user?distinct_id=distinct_id_0")
+        distinct_id_0_user = User.objects.create_and_join(self.organization, "distinct_id_0_user@posthog.com", None)
+        distinct_id_0_user.distinct_id = "distinct_id_0"
+        distinct_id_0_user.save()
+        self.client.force_login(distinct_id_0_user)
+        response = self.client.get("/api/feature_flag/my_flags")
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(response.json()["distinct_id"], "distinct_id_0")
-        self.assertEqual(sorted(response.json()["flags_enabled"]), ["red_button"])
+        self.assertEqual(response.json()["flags"], {"red_button": True})
 
-        # error if no distinct_id
-        response = self.client.get("/api/feature_flag/for_user?distinct_id=")
+    def test_create_override(self):
+        response = self.client.get("/api/feature_flag_override")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(response.json()["results"]), 0)
+
+        # Boolean override value
+        feature_flag_instance = FeatureFlag.objects.create(team=self.team, created_by=self.user, key="beta-feature")
+        response = self.client.post(
+            "/api/feature_flag_override",
+            {"feature_flag": feature_flag_instance.id, "user": self.user.id, "override_value": True},
+        )
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+        # String override value
+        feature_flag_instance_2 = FeatureFlag.objects.create(team=self.team, created_by=self.user, key="beta-feature-2")
+        response = self.client.post(
+            "/api/feature_flag_override",
+            {"feature_flag": feature_flag_instance_2.id, "user": self.user.id, "override_value": "hey hey hey"},
+        )
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+        response = self.client.get("/api/feature_flag_override")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(
+            len(response.json()["results"]), 2,
```

Comment:
You don't actually assert the objects were created properly, i.e. right  FF & values are stored.